lib/ts/recipe/multitenancy/types.ts

```diff
@@ -0,0 +1,79 @@
+import type { UserInput as RecipeModuleUserInput } from "../recipeModule/types";
+import type OverrideableBuilder from "supertokens-js-override";
+import type { RecipeFunctionOptions } from "supertokens-web-js/recipe/recipeModule/types";
+
+export type PreAndPostAPIHookAction = "GET_LOGIN_METHODS";
+
+export type UserInput = {
+    override?: {
+        functions?: (
+            originalImplementation: RecipeInterface,
+            builder: OverrideableBuilder<RecipeInterface>
+        ) => RecipeInterface;
+    };
+    getTenantID?: (input?: { userContext: any }) => Promise<string> | string | undefined;
```

Comment:
Should change to getTenantId. Cause in the backend node SDK, it's getTanantId with a small "d"